packages/code-display/src/utils/utils.ts

```diff
@@ -0,0 +1,38 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2021 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+import { Language } from 'prism-react-renderer'
+
+export const getPrismLanguage = (language: string): Language => {
+  language = language.toLowerCase()
+  const unstyled = ['kotlin', 'csharp', 'swift']
+  // TODO revert back to `go` in generator language definitions instead of using this
+  if (language === 'golang') {
+    return 'go'
```

Comment:
nit: consider having an enum/hash that maps our SDK languages to "prism languages" to avoid having to grow this if statement as our set of supported SDK languages grows

Reply:
Better to just change the language definition back to `go` from `golang` in sdk-codegen and this hack won't be needed

Reply:
Do you have an example of an enum to enum mapping? 

I think most of the languages will be the same, as "go" was "golang" until it was changed for Ace. Once the other highlighers are brought in & we change golang back, this function becomes:
```
export const getPrismLanguage = (language: string): Language => {
  return language.toLowerCase() as Language
}
```
Which is an enum to enum mapper?

Reply:
codeGenerators.ts is where the languages are defined. If Ace is only left for the form editor now, that's json, so we don't need to worry about how Ace would highlight 'go'.  The cast should work fine depending on how Prism defines it.